src/page/GroupCreate.tsx

```diff
@@ -68,13 +71,39 @@ const GroupCreate = () => {
     setIsActive(allFieldsFilled || (isPublicGroup && allFieldsFilled));
   }, [inputs, isPublicGroup]);
 
-  const handleGroupCreate = () => {
-    navigate('/group-complete', {
-      state: {
-        groupPassword: inputs.secretKey,
-        thumbnailUrl: previewImage as string,
-      },
-    });
+  const handleGroupCreate = async () => {
+    const postData = {
+      title: inputs.title,
+      password: inputs.secretKey,
+      capacity: inputs.num,
+      tags: selectedTags,
+      introduce: inputs.intro,
+      information: inputs.group,
+    };
```

Comment:
P2: 이 부분 inputs.가 반복되면서 가독성이 저하되는 것 같아요
코드 상단(변수 선언부)에서 `const { title, password, capacity, introduce, information } = inputs;` 라고 정의해두면 아래 형식으로 쓸 수 있습니당
```suggestion
    const postData = {
      title: title,
      password: secretKey,
      capacity: num,
      tags: selectedTags,
      introduce: intro,
      information: group,
    };
```

Reply:
객체분해가 아직 익숙치가 않아서 생각은 했는데 뭐랄까,, 많이 사용해보지 않아서 아직 미숙하네요 ! 감사합니다